CONTRIBUTING.md

```diff
@@ -1,10 +1,13 @@
 # Contribution guidelines
 
+We welcome contributions from the open source community.
 If you want to contribute, we ask you to follow these guidelines.
-These instructions are currently mainly intended for our own Lifely developers, but
-we do also welcome contributions from the open source community.
 Please reach out to us if you have questions or wish to contribute.
```

Comment:
```suggestion
Please reach out to [us](email:zaakafhandelcomponent@info.nl) if you have questions or wish to contribute.
```

Reply:
Or leave it up to individuals to find a contributor's contact details?

Reply:
I would point them to https://github.com/infonl/dimpact-zaakafhandelcomponent/discussions and leave it at that.

Reply:
I did the latter now.